src/main/java/net/neoforged/neoforge/event/level/BlockEvent.java

```diff
@@ -54,49 +52,23 @@ public BlockState getState() {
     }
 
     /**
-     * Event that is fired when an Block is about to be broken by a player
+     * Event that is fired when a Block is about to be broken by a player
      * Canceling this event will prevent the Block from being broken.
      */
     public static class BreakEvent extends BlockEvent implements ICancellableEvent {
-        /** Reference to the Player who broke the block. If no player is available, use a EntityFakePlayer */
+        /**
+         * Reference to the Player who broke the block. If no player is available, use a EntityFakePlayer
```

Comment:
"If no player is available"?

Reply:
I agree that this is a bit weird, this event is only supposed to be fired from player block breaking, so if no player is involved the event just does not fire, correct? Fake player only makes sense if you wish to mimic a player breaking a block.

Reply:
That's just a formatting change; not sure where the original wording came from.  Given that it references `EntityFakePlayer`, it's _really_ old.

Reply:
Updated the docs to something from this decade.